Bash completion step fails during compile

Everytime I try to build pakku, either from Git or from Aur, I get the following error:

GEN: completion/bash
1 out of 4 hunks FAILED
make: *** [Makefile:90: completion/bash] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
Error making: pakku

--- a/completion/bash.patch
+++ b/completion/bash.patch
@@ -1,19 +1,19 @@
 --- a/bash
 +++ b/bash
-@@ -50,2 +50,3 @@
+@@ -117,2 +117,3 @@
 -         info list needed nodeps assume-installed print refresh recursive search sysupgrade'
 -        'c g i l p s u w y')
 +         info list needed nodeps assume-installed print refresh recursive search sysupgrade
 +         build keyserver noaur source'
 +        'c g i l n p s u w y z')
-@@ -64,1 +65,1 @@
+@@ -131,1 +132,1 @@
 -    ! $prev = --@(cachedir|color|config|dbpath|help|hookdir|gpgdir|logfile|root|version) ]]
 +    ! $prev = --@(cachedir|color|config|dbpath|help|hookdir|gpgdir|keyserver|logfile|root|version) ]]
-@@ -81,1 +82,1 @@
+@@ -151,1 +152,1 @@
 -          _pacman_pkg Slq;;
 +          _arch_compgen "`pakku -Ssq "$cur" 2> /dev/null`";;
-@@ -93,3 +94,1 @@
--complete -F _pacman -o default pacman
--complete -F _makepkg -o default makepkg
+@@ -164,3 +165,1 @@
+-complete -F _pacman pacman
+-complete -F _makepkg makepkg
 -complete -F _pacman_key -o default pacman-key
 +complete -F _pakku -o default pakku

